### PR TITLE
Added code for 2 sort algos

### LINKED ISSUE #17: Add Merge and Selection sort algorithm codes

### LINKED ISSUE #20: Missing function call in Merge Sort

### LINKED ISSUE #21: Missing main function in Selection Sort

### PR DESCRIPTION
**Pull request by shri-platypus**

### CODE REVIEW

File: javaCode/src/sortAlgos/MergeSort.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+}
+void sort(int arr[], int beg, int end)
+{
+if (beg<end)
+{
+int mid = (beg+end)/2;
+sort(arr, beg, mid);
+sort(arr , mid+1, end);
+
```
Comment (Shwetannaa) [inline javaCode/src/sortAlgos/MergeSort.java]: Missing function call #20

**Review by Shwetannaa — CHANGES_REQUESTED** [review]
Missing function call

File: javaCode/src/sortAlgos/SelectionSort.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                    index = j;//searching for lowest index  
+                }  
+            }  
+            int smallerNumber = arr[index];   
+            arr[index] = arr[i];  
+            arr[i] = smallerNumber;  
+        }  
+    }  
+       
+
```
Comment (soumyachan) [inline javaCode/src/sortAlgos/SelectionSort.java]: Missing main function #21

**Review by soumyachan — CHANGES_REQUESTED** [review]
Missing main function in Selection Sort

File: javaCode/src/sortAlgos/MergeSort.java (hunk 1 above)
Comment (shri-platypus) [reply javaCode/src/sortAlgos/MergeSort.java]: Fixed the issue #20

File: javaCode/src/sortAlgos/SelectionSort.java (hunk 2 above)
Comment (shri-platypus) [reply javaCode/src/sortAlgos/SelectionSort.java]: Fixed the issue #21

**Review by Shwetannaa — APPROVED** [review]
Fixed issue

**Review by soumyachan — APPROVED** [review]
Issues is fixed

**Pull request merged**